src/theme/_fonts.scss

```diff
@@ -3,6 +3,17 @@
   --ion-font-family: "Roboto", "Helvetica Neue", sans-serif;
 }
 
+// Override roboto font
+:root {
+  --ion-font-family: "Nunito", sans-serif;
```

Comment:
nit (blocking): It would be good to move this (and the `@font-face` declaration below) to the `plh_kids_kw` theme file. Whilst we may well want to incorporate these changes into the default styles, if we keep them scoped to the `plh_kids_kw` theme for now then we avoid making changes to our existing production apps, and we can decide if/when to roll out the changes independently. 

Ideally, it would be good to be able to expose a new variable for `$font-name` (alongside `$color-primary`, `$color-secondary` and `$page-background` that already exist at the top of the theme files), and infer all the necessary changes from that. However I think that could be tricky as we'd need to account for different font types (we already have `.ttf` and `.woff2` to handle). 

So I think for now it might make sense to do the following:
1. Move the whole `@font-face` declaration block as it is into the theme file (i.e. under line 10 of `src/theme/themes/plh_kids_kw.scss`)
2. Move this variable assignment to the theme file
    - Adding `ion-font-family: "Nunito", sans-serif,` to the list of `$variable-overrides` should work I think – the `--` gets added automatically

Let me know if that doesn't make sense

Reply:
@jfmcquade The scoping of changes does make sense. Given that the font change was required globally, it made sense for me to have that in the variables file but I do understand how that makes deployment somewhat complicated.
I'll work on keeping changes within the theme file.